hal/src/rtl872x/usbd_driver.h

```diff
@@ -116,7 +116,7 @@ class RtlUsbDriver : public DeviceDriver {
         .speed = USB_SPEED_HIGH_IN_FULL,
         .dma_enable = 0, // ?
         .self_powered = 1,
-        .isr_priority = RTL_USBD_ISR_PRIORITY,
+        .isr_priority = rtl::RTL_USBD_ISR_PROCESSING_THREAD_PRIORITY,
```

Comment:
Did we just want to move `RTL_USBD_ISR_PROCESSING_THREAD_PRIORITY` into the `rtl` namespace to move it somewhere more appropriate? Are we worried about people using it to assign their own thread priorities?

Reply:
Just moved it out of the class, renamed to be more explicit of what it is (not ISR priority, but a task priority) and there was a somewhat appropriate namespace already with some other constant, so just put it in there.